build/NuGetPackages/Microsoft.Build.Runtime.nuspec

```diff
@@ -90,5 +91,6 @@
     <file src="$outputPathNetCore$Microsoft.VisualStudioVersion.v11.Common.props" target="contentFiles\any\netcoreapp1.0\" buildAction="None" copyToOutput="true"/>
     <file src="$outputPathNetCore$Microsoft.VisualStudioVersion.v12.Common.props" target="contentFiles\any\netcoreapp1.0\" buildAction="None" copyToOutput="true"/>
     <file src="$outputPathNetCore$Microsoft.VisualStudioVersion.v14.Common.props" target="contentFiles\any\netcoreapp1.0\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPathNetCore$NuGet.MSBuildSdkResolver.dll" target="contentFiles\any\netcoreapp1.0\SdkResolvers\NuGet.MSBuildSdkResolver\" buildAction="None" copyToOutput="true"/>
```

Comment:
Putting them in this package means that if people reference only the Microsoft.Build package to evaluate projects, they won't get this resolver. Is this intended?

Reply:
Yes, evaluation can only be done by referencing `Microsoft.Build.Runtime`